types/mui-datatables/mui-datatables-tests.tsx

```diff
@@ -45,6 +48,12 @@ const MuiCustomTable: React.FC<Props> = props => {
         {
             name: 'amount',
             label: 'Amount',
+            options: {
+                customHeadLabelRender: (dataIndex: number, rowIndex: number) => {
+                    console.log(`custom header label render for dataIndex ${dataIndex} and rowIndex ${rowIndex}`);
```

Comment:
```suggestion
```

No console logs please

Reply:
Thanks for your quick Review and apologize for some rookie mistakes, pretty much my first PR here :)
I did it because I saw some console logs on "onTableChange".

Reply:
Ah well those are not in the scope of the PR right now but if you want to you can remove those as well